web/geo/generateExchangesToExclude.ts

```diff
@@ -1,5 +1,5 @@
-import { mergeExchanges } from '../scripts/generateZonesConfig';
-import { fileExists, getJSON, writeJSON } from './utilities';
+import { mergeExchanges } from '../scripts/generateZonesConfig.js';
+import { fileExists, getJSON, writeJSON } from './utilities.js';
```

Comment:
Why `.js` here when source is in TS? 🤔

Reply:
This is part of the cascading issue that caused so many seemingly unrelated changes.
First using the meta imports forced the us to use pure js modules and set the flag in the package.json.
This in turn forced us/me to add extensions to the files that aren't used in the bundler.
Since we use typescript and tsc to compile these it would be logical if they where ts files but tsc only changes typescript related code and don't touch the imports (unless they are marked as types) which means that we have to use the js extension because it will be js when it's compiled and run.

It's all a little messy but should only be a one time change. Hopefully 😅